node/hyperbahn/index.js

```diff
@@ -425,3 +435,35 @@ HyperbahnClient.prototype.destroy = function destroy() {
     timers.clearTimeout(self._advertisementTimer);
     timers.clearTimeout(self.advertisementTimeoutTimer);
 };
+
+function safeSyncRead(filePath) {
+    var fileContents;
```

Comment:
please initialize these to null rather than setting object fields to undefined implicitly